docs/source/internals/getting_started.rst

```diff
@@ -260,6 +227,39 @@ Check your database settings. A quick way to get started is to use SQLite:
 Note that we recommend using ``ATOMIC_REQUESTS`` to tie transactions to
 requests.
 
+URLs
+====
+
+Alter your ``frobshop/urls.py`` to include Oscar's URLs. You can also include
+the Django admin for debugging purposes. But please note that Oscar makes no
+attempts at having that be a workable interface; admin integration exists
+to ease the life of developers.
+
+If you have more than one language set your Django settings for ``LANGUAGES``,
+you will also need to include Django's i18n URLs:
+
+.. code-block:: django
+
+    from django.apps import apps
+    from django.conf.urls import include, url  # < Django-2.0
+    # from django.urls import include, path  # > Django-2.0
+    from django.contrib import admin
+
+    urlpatterns = [
+        # url(r'^i18n/', include('django.conf.urls.i18n')), # < Django-2.0
```

Comment:
Could you please remove the `< Django-2.0` comments entirely? We have dropped support for Django 2.1 and lower.